handout/architect/10_ways_to_misuse_angular.md

```diff
@@ -0,0 +1,103 @@
+# Ten Ways to Misuse Angular
+
+1. Ignoring a component’s host element
```

Comment:
Just checking, do I understand correctly that the main point is what *not* to do, and the sub-points are what people *should* do?